web/src/message_edit.js

```diff
@@ -802,7 +802,7 @@ export function save_inline_topic_edit($row) {
     const message = message_lists.current.get(message_id);
 
     const old_topic = message.topic;
-    const new_topic = $row.find(".inline_topic_edit").val();
+    const new_topic = $row.find("#inline_topic_edit").val();
```

Comment:
I think actually the correct thing to do is get rid of the IDs, since they're duplicates in any interleaved view with multiple recipient bars, and keep the class.